### PR TITLE
TIP-713-fix-quick-export-grid-context: fix quick export grid context

### PR DESCRIPTION
**Pull request by momoss**
| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: features/mass-action/quick_export_grid_context.feature [hunk 1]
```diff
@@ -33,10 +33,10 @@ Feature: Quick export products according to the product grid context
     And the name of the exported file of "csv_product_grid_context_quick_export" should be "products_export_grid_context_en_US_tablet.csv"
     And exported file of "csv_product_grid_context_quick_export" should contain:
     """
-    sku;color;family;groups;name-en_US;price-EUR;price-USD;size;description-en_US-tablet;weight;weight-unit
-    boots;black;boots;;Amazing boots;20;25;40;Mob;20;GRAM
-    sneakers;white;sneakers;;Sneakers;50;60;42;ylette;4;GRAM
-    pump;blue;Pump;;;15;20;41;;;
+    sku;color;description-en_US-tablet;family;groups;name-en_US;price-EUR;price-USD;size;weight;weight-unit
```
Comment (SamirBoulil) [inline features/mass-action/quick_export_grid_context.feature:36]: Are we sure don't break the functionnal here ? Isn't there some rule about column order ?

File: src/Pim/Bundle/EnrichBundle/Connector/Processor/QuickExport/ProductProcessor.php [hunk 2]
```diff
@@ -97,7 +98,12 @@ public function process($product)
         $productStandard = $this->normalizer->normalize($product, 'standard', $normalizerContext);
 
         if ($this->areAttributesToFilter($parameters)) {
-            $productStandard = $this->filterProperties($productStandard, $parameters->get('selected_properties'));
+            $selectedProperties = $parameters->get('selected_properties');
+            if (in_array('identifier', $selectedProperties)) {
+                $identifer = $this->attributeRepository->findOneBy(['type' => AttributeTypes::IDENTIFIER]);
+                $selectedProperties = array_merge([$identifer->getCode()], $selectedProperties);
```
Comment (SamirBoulil) [inline src/Pim/Bundle/EnrichBundle/Connector/Processor/QuickExport/ProductProcessor.php]: You could have used http://php.net/manual/en/function.array-unshift.php if you wanted also.

**Review by SamirBoulil — APPROVED** [review]
GTM if it does not break the column order functionnal

File: src/Pim/Bundle/EnrichBundle/Connector/Processor/QuickExport/ProductProcessor.php (hunk 2 above)
Comment (momoss) [reply src/Pim/Bundle/EnrichBundle/Connector/Processor/QuickExport/ProductProcessor.php]: Yes thank's

File: features/mass-action/quick_export_grid_context.feature (hunk 1 above)
Comment (momoss) [reply features/mass-action/quick_export_grid_context.feature:36]: In fact, if you read the data, I don't understand how the test can be green

**Review by jjanvier — APPROVED** [review]

File: features/mass-action/quick_export_grid_context.feature [hunk 3]
```diff
@@ -33,10 +33,10 @@ Feature: Quick export products according to the product grid context
...
+    boots;black;Mob;boots;;Amazing boots;20;25;40;20;GRAM
+    sneakers;white;ylette;sneakers;;Sneakers;50;60;42;4;GRAM
+    pump;blue;;;;Pump;15;20;41;;
```
Comment (damien-carcel) [inline features/mass-action/quick_export_grid_context.feature:39]: 😆

**Review by damien-carcel — APPROVED** [review]

**Comment by momoss** [comment]
@SamirBoulil It's alphabetical ordered

**Pull request merged**